gCNV/cnv_germline_case_filter_workflow.wdl

```diff
@@ -341,11 +342,17 @@ task ExtractPoNFreq {
                 df[f] = df['SAMPLE'].str.split(':').apply(lambda x: x[num])
             return df
 
+        def get_sample_id(vcf):
+            with gzip.open(vcf,"r") as f_vcf:
```

Comment:
For future reference, you can use `rt` instead of `r` to read the file as text, then you don't need to decode it.